### PR TITLE
Improve CI

### PR DESCRIPTION
**Pull request by szepeviktor**
... by removing inspirational comments.

### CODE REVIEW

File: .github/workflows/integrate.yml [hunk 1]
```diff
@@ -36,11 +36,10 @@ jobs:
 
       - name: "Check file permissions"
         run: |
-          test "$(find . -type f -not -path './.git/*' -not -path './tests/integration/bin/*' -executable)" = ""
+          test "$(find . -type f -not -path './.git/*' -executable)" = "./tests/integration/bin/install-wp-tests.sh"
```
Comment (szepeviktor) [inline .github/workflows/integrate.yml:39]: The whole point of `""` is to list executables.

File: .github/workflows/integrate.yml [hunk 2]
```diff
@@ -105,15 +103,14 @@ jobs:
...
-#        run: |
-#          wget "https://github.com/php-coveralls/php-coveralls/releases/download/v2.5.2/php-coveralls.phar"
-#          php ./php-coveralls.phar -v
+      - name: "Send coverage to Coveralls"
+        env:
+          COVERALLS_REPO_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        if: "${{ env.COVERALLS_REPO_TOKEN && matrix.php-version == '8.1' && matrix.dependencies == 'highest' }}"
+        run: |
+          wget "https://github.com/php-coveralls/php-coveralls/releases/download/v2.5.2/php-coveralls.phar"
+          php ./php-coveralls.phar -v
```
Comment (szepeviktor) [inline .github/workflows/integrate.yml:113]: Now we have coverage!!

**Comment by szepeviktor** [comment]
@chesio Please help me! When to include stubs?

File: .github/workflows/integrate.yml (hunk 2 above)
Comment (szepeviktor) [reply .github/workflows/integrate.yml:117]: There must be a better way than including the stubs of `WP_List_Table`.

File: .github/workflows/integrate.yml (hunk 1 above)
Comment (chesio) [reply .github/workflows/integrate.yml:39]: I see now, thanks :)

**Comment by chesio** [comment]
Thanks @szepeviktor!

Btw. I didn't mind the inspirational comments ;-)

**Pull request merged**